### PR TITLE
Add Linkedin notification

### PR DESCRIPTION
**Pull request by Peltoche**

### CODE REVIEW

File: server/konnectors/linkedin.coffee [hunk 1]
```diff
@@ -305,7 +327,7 @@ saveContact  = (newContact, entries, callback) ->
 
     # Case where the contact already exists and where it was imported from
     # Linkedin.
-    if entries.cozyContactsByAccountIds[newContact.id]?
+    if entries.cozyContactsByAccountIds[linkAccount.id]?
```
Comment (frankrousseau) [inline server/konnectors/linkedin.coffee]: You should keep newContact here.

Comment (Peltoche) [inline server/konnectors/linkedin.coffee]: The array cozyContactsByAccountIds references each contact by its account id and not the contact id. The last commit (newContact.id) leades to re-update all contacts each time.

Comment (frankrousseau) [inline server/konnectors/linkedin.coffee]: Oups sorry I didn't notice the linkAccount variable was declared before.

**Comment by frankrousseau** [comment]
Thx!

**Pull request merged**